Northstar.Client/mod/scripts/vscripts/ui/menu_mod_settings.nut

```diff
@@ -0,0 +1,1111 @@
+		})
+		Hud_AddEventHandler( child, UIE_LOSE_FOCUS, void function( var child ) : (panel)
+		{
+			Hud_SetColor( Hud_GetChild( panel, "ResetModImage" ), 180, 180, 180, 180 )
+		})
+
+		child.SetNavUp( Hud_GetChild( file.modPanels[ int( PureModulo( i - 1, len ) ) ], "ResetModToDefault" ) )
+		child.SetNavDown( Hud_GetChild( file.modPanels[ int( PureModulo( i + 1, len ) ) ], "ResetModToDefault" ) )
+
+		Hud_AddEventHandler( child, UIE_CLICK, ResetConVar )
+		file.resetModButtons.append(child)
+		//Hud_AddEventHandler( Hud_GetChild( panel, "ResetModImage" ), UIE_CLICK, ResetConVar )
+
+		// text field nav
+		child = Hud_GetChild( panel, "TextEntrySetting" )
+
+		//
+		Hud_AddEventHandler( child, UIE_LOSE_FOCUS, SendTextPanelChanges )
+
+		child.SetNavUp( Hud_GetChild( file.modPanels[ int( PureModulo( i - 1, len ) ) ], "TextEntrySetting" ) )
+		child.SetNavDown( Hud_GetChild( file.modPanels[ int( PureModulo( i + 1, len ) ) ], "TextEntrySetting" ) )
+
+		child = Hud_GetChild( panel, "Slider" )
+
+		child.SetNavUp( Hud_GetChild( file.modPanels[ int( PureModulo( i - 1, len ) ) ], "Slider" ) )
+		child.SetNavDown( Hud_GetChild( file.modPanels[ int( PureModulo( i + 1, len ) ) ], "Slider" ) )
+
+		file.sliders.append( MS_Slider_Setup( child ) )
+
+		Hud_AddEventHandler( child, UIE_CHANGE, OnSliderChange )
+
+		child = Hud_GetChild( panel, "OpenCustomMenu" )
+
+		Hud_AddEventHandler( child, UIE_CLICK, CustomButtonPressed )
+	}
+
+	// Hud_AddEventHandler( Hud_GetChild( file.menu, "BtnModsSearch" ), UIE_LOSE_FOCUS, OnFilterTextPanelChanged )
+	Hud_AddEventHandler( Hud_GetChild( file.menu, "BtnFiltersClear" ), UIE_CLICK, OnClearButtonPressed )
+	// mouse delta
+	AddMouseMovementCaptureHandler( file.menu, UpdateMouseDeltaBuffer )
+
+	Hud_AddEventHandler( Hud_GetChild( file.menu, "BtnModsSearch" ), UIE_CHANGE, void function ( var inputField ) : ()
+	{
+		file.filterText = Hud_GetUTF8Text( inputField )
+		OnFiltersChange(0)
+	} )
+}
+
+// "PureModulo"
+// Used instead of modulo in some places.
+// Why? beacuse PureModulo loops back onto positive numbers instead of going into the negatives.
+// DO NOT TOUCH.
+// a / b != floor( float( a ) / b )
+// int( float( a ) / b ) != floor( float( a ) / b )
+// Examples:
+// -1 % 5 = -1
+// PureModulo( -1, 5 ) = 4
+float function PureModulo( int a, int b )
+{
+	return b * ( ( float( a ) / b ) - floor( float( a ) / b ) )
+}
+
+void function ResetConVar( var button )
+{
+	print("ok")
+	ConVarData conVar = file.filteredList[ int ( Hud_GetScriptID( Hud_GetParent( button ) ) ) + file.scrollOffset ]
+
+	if ( conVar.isCategoryName )
+	{
+		ShowAreYouSureDialog( "#ARE_YOU_SURE", ResetAllConVarsForModEventHandler( conVar.catName ), "#WILL_RESET_ALL_SETTINGS"  )
+	}
+	else ShowAreYouSureDialog( "#ARE_YOU_SURE", ResetConVarEventHandler( int ( Hud_GetScriptID( Hud_GetParent( button ) ) ) + file.scrollOffset ), Localize( "#WILL_RESET_SETTING", Localize( conVar.displayName ) ) )
```

Comment:
please be consistent with formatting your conditionals at least within the same ladder